### ISSUE TITLE
feat: Exclude a package from update

### ISSUE THREAD

**Issue by mbevc1** [marcosnils/bin#166]
It would be useful to have an exclude flag when updating in order to skip packages you don't want to update. It's the opposite of updating a single package and perhaps slightly adjecent to issue we have for version pinning #126 :)

e.g. `bin update -x /some/path/binary`

**Comment by marcosnils** [comment]
yep, I like it. Additionally we could introduce a similar concept of `apt-mark hold` which persists this config in case you don't want to specify it each time.

**Comment by mbevc1** [comment]
Sounds good :+1:

**Comment by mbevc1** [comment]
If I find some time I'll try looking into it and see if there is anything quick we could do here :)
